uvm/impl/com/untangle/uvm/UvmContextImpl.java

```diff
@@ -1195,6 +1197,18 @@ public String getHelpUrl()
         return url;
     }
 
+    /**
+     * getFeedbackUrl - get the feedback URL for the UI
+     * @return String
+     */
+    public String getFeedbackUrl()
+    {
+        String url = System.getProperty(PROPERTY_FEEDBACK_URL);
+        if (url == null)
```

Comment:
@cblaise-untangle don't we need do an empty/blank string test instead of just a null check ?

Reply:
I believe System.gerProperty returns null if the value is not found.  But check to see if it does return a blank string and if so, yes, use that conditional.

Reply:
@mayur-dhande @cblaise-untangle I confirmed that System.getProperty returns null if value not found. only null check should be fine.